docs/kb/rules/understand-severities.md

```diff
@@ -0,0 +1,20 @@
+---
+description: Understand how rule severity is determined.
+tags:
+  - Rules
+  - Semgrep Registry
+---
+
+# How does Semgrep assign severity levels to rules?
+
+## Semgrep Code
+
+Semgrep Code rules have one of three severity levels: ERROR (High), WARNING (Medium), or INFO (Low). The severity indicates how critical the issues are that a rule potentially detects.
+
+The severity of a given rule is assigned by the rule author. For custom and third-party rules, the rule author's assignment is the source of truth.
```

Comment:
```suggestion
The severity of a given rule is assigned by the rule author. For custom and third-party rules, the rule author is the source of truth for the severity.
```

Reply:
I was initially a little confused by the wording, does this sound any better?

Reply:
Huh, I thought it made sense - the severity is assigned by the author, and that assignment is the source of truth. Though I probably shouldn't be writing in passive voice anyway!

What about:

> The rule author assigns the rule severity. For custom and third-party rules, their severity assignment is the source of truth.